pkg/controller/serviceaccount/serviceaccounts_controller.go

```diff
@@ -202,13 +202,11 @@ func (c *ServiceAccountsController) syncNamespace(ctx context.Context, key strin
 
 	createFailures := []error{}
 	for _, sa := range c.serviceAccountsToEnsure {
-		switch _, err := c.saLister.ServiceAccounts(ns.Name).Get(sa.Name); {
-		case err == nil:
-			continue
-		case apierrors.IsNotFound(err):
-		case err != nil:
+		if _, err := c.saLister.ServiceAccounts(ns.Name).Get(sa.Name); err != nil {
 			return err
 		}
+		continue
```

Comment:
This continue line at the end of this switch case means all code below the switch is dead code.